### ISSUE TITLE
RingTone on Sd Card doesn't work while connected in USB

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#65]
```
1. Set a ringtone located on SD Card
2. connect the usb cable with Access SD Card Mode
3. call ring command

What is the expected output? What do you see instead?
The phone give an error message because the ringtone is innacessible.
In this case the use of the default ringtone would be better like Android do it.

What version of the product are you using? On what operating system?
1.7.3

Please provide any additional information below.
- I use a Motorola Milestone (knownk as Droid in USA)
- I use aRing app to set a MP3 as ringtone
- The chat :
moi:  ring:95
 Tel:  Le téléphone sonne
 Impossible de faire sonner le téléphone, vérifier les paramètres.




```

Original issue reported on code.google.com by `esteve.g...@gmail.com` on 28 Jan 2011 at 1:05

**Comment by GoogleCodeExporter** [comment]
```
small patch.

Background: When the Phone is connected to USB AND the SD-Storage is transfered 
to the pc, the contents of the SD-Storage are no more accessible for the phone 
as the SD-Storage is now a physical volume of the PC.
```

Original comment by `fschm...@gmail.com` on 29 Jan 2011 at 3:18
- Changed state: **Started**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

Attachments:
- [ringtone.bundle](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-65/comment-1/ringtone.bundle)

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 29 Jan 2011 at 4:01
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 29 Jan 2011 at 7:02

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.8
```

Original comment by `Florent....@gmail.com` on 29 Jan 2011 at 10:57
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
